### DISCUSSION TITLE
Vietnamese searching issue

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#2391] [Q&A]
My issue related UTF-8 Vietnamese language
My title: ĐÀO TẠO KHOÁ HỌC cho sinh viên

if i use mysql, I can search by keyword: `dao tao` or lowercase `đào tạo`

But with SQLite, I can not.. I have to type `ĐÀO TẠO` (I mean it has to search by UPPERCASE keyword)... This is really inconvenient

Could you give me some Idea ? This is very important :((

**Comment by ganigeorgiev** [comment]
I think this is related to https://github.com/pocketbase/pocketbase/issues/627.

You can try to build your custom PocketBase with the CGO driver and ICU extension enabled, something like:

```sh
CGO_ENABLED=1 go build -tags "sqlite_icu"

# start the generated executable as usual
./myapp serve
```

**Reply by hungcrush** [reply]
Thank you so much. I will try
